examples/simple_dashboard/image_resources.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -e
```

Comment:
File name doesnt match the name of the file to run, which will cause it to fail

Reply:
fixed on #39474a8e